GDJS/Runtime/runtimescene.js

```diff
@@ -156,6 +144,18 @@ gdjs.RuntimeScene.prototype.loadFromScene = function(sceneData) {
     this._timeManager.reset();
 };
 
+/**
+ * Registration of {@link gdjs.RuntimeObject}. 
+ * This step is needed for any object before adding Instances of it to the scene.
```

Comment:
I would go for:
`Call this function before adding instances of an object to the scene.`